portfolio.html

```diff
@@ -181,14 +190,146 @@ <h2 class="name">Multy-Post Stories</h2>
+      </div>
+      <div class="infoCard">
+      <section class="info">
+        <h2 class="name">Multy-Post Stories</h2>
+        <ul class="info_job">
+          <li class="job">Facebook</li>
+          <li>FULL Stack Dev</li>
+          <li>2015</li>
+        </ul>
+      </section>
+      <section class="decription">
+        <p>
+          Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+        </p>
+      </section>
+      <section class="linkButtons">
+        <button class="link">html</button>
+        <button class="link">Ruby on rails</button>
+        <button class="link">css</button>
+        <button class="link">javaScript</button>
+      </section>
+      <div class="seeMore">
+        <button class="more">See Project</button>
+      </div>
+    </div>
+  </div>
+ 
+   <!-- Fourth page  -->
+ 
+ <div class="desktop3">
+  <div class="img">
+    <img class="imgBG" src="img/d3.png" alt="portfolio card image"></img>
+  </div>
+   <div class="infoCard odd">
+   <section class="info">
+     <h2 class="name">Facebook 360</h2>
+     <ul class="info_job">
+       <li class="job">FACEBOOK</li>
+       <li>Full Stack Dev</li>
+       <li>2015</li>
+     </ul>
+   </section>
+   <section class="decription">
+     <p>
+      Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
+     </p>
+   </section>
+   <section class="linkButtons">
+     <button class="link">html</button>
+     <button class="link">Ruby on rails</button>
+     <button class="link">css</button>
+     <button class="link">javaScript</button>
+   </section>
+   <div class="seeMore">
+     <button class="more">See Project</button>
+   </div>
+ </div>
+ </div>
+ 
+ <!-- Fifth page -->
+ 
+ <div class="desktop4">
+  <div class="img">
+    <img class="imgBG" src="img/d4.png" alt="portfolio card image"></img>
+  </div>
+   <div class="infoCard">
+   <section class="info">
+     <h2 class="name">Uber Navigation</h2>
+     <ul class="info_job">
+       <li class="job">Uber</li>
+       <li>Lead Developer</li>
+       <li>2018</li>
+     </ul>
+   </section>
+   <section class="decription">
+     <p>
+      A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.
+     </p>
+   </section>
+   <section class="linkButtons">
+     <button class="link">html</button>
+     <button class="link">Ruby on rails</button>
+     <button class="link">css</button>
+     <button class="link">javaScript</button>
+   </section>
+   <div class="seeMore">
+     <button class="more">See Project</button>
+   </div>
+ </div>
+ </section>
+ </div>
+ </section>
+</section>
 
     <!-- Portfolio "about me" section -->
     <!-- ABOUT ME 1 PAGE -->
     <section class="mainContainerAboutme">
       <header class="top">
```

Comment:
- [ ] I think it's not recommended to use header as a common container inside a section, Kindly replace it with another container or remove it completely :recycle: to make it semantically correct :+1: